Dockerfile

```diff
@@ -50,13 +50,13 @@ RUN SECRET_KEY=dummy STATIC_URL=${STATIC_URL} python3 manage.py collectstatic --
 EXPOSE 8000
 ENV PYTHONUNBUFFERED=1
 
-LABEL org.opencontainers.image.title="saleor/saleor"                                  \
-      org.opencontainers.image.description="\
-A modular, high performance, headless e-commerce platform built with Python, \
-GraphQL, Django, and ReactJS."                                                         \
-      org.opencontainers.image.url="https://saleor.io/"                                \
-      org.opencontainers.image.source="https://github.com/saleor/saleor"               \
-      org.opencontainers.image.authors="Saleor Commerce (https://saleor.io)"           \
-      org.opencontainers.image.licenses="BSD 3"
-
-CMD ["gunicorn", "--bind", ":8000", "--workers", "4", "--worker-class", "saleor.asgi.gunicorn_worker.UvicornWorker", "saleor.asgi:application"]
+LABEL org.opencontainers.image.title="saleor/saleor" \
+  org.opencontainers.image.description=" \
+  A modular, high performance, headless e-commerce platform built with Python, \
+  GraphQL, Django, and ReactJS." \
+  org.opencontainers.image.url="https://saleor.io/" \
+  org.opencontainers.image.source="https://github.com/saleor/saleor" \
+  org.opencontainers.image.authors="Saleor Commerce (https://saleor.io)" \
+  org.opencontainers.image.licenses="BSD-3-Clause"
```

Comment:
This is a cosmetic change; "BSD-3-Clause" is the canonical code for our license.